### PR TITLE
Updating our jobs to use the VS2015.3 image.

### PR DESCRIPTION
**Pull request by tannergooding**
FYI. @jasonmalinowski, @jaredpar, @dotnet/roslyn-infrastructure

### CODE REVIEW

**Comment by tannergooding** [comment]
test ci please

File: netci.groovy [hunk 1]
```diff
@@ -103,7 +103,7 @@ set TMP=%TEMP%
 
       def triggerPhraseOnly = configuration == 'release'   
       def triggerPhraseExtra = ""
-                Utilities.setMachineAffinity(myJob, 'Windows_NT', 'latest-or-auto')
+                Utilities.setMachineAffinity(myJob, 'Windows_NT', 'latest-or-auto-update3')
```
Comment (jaredpar) [inline netci.groovy]: This has the words "latest" and "update3" which seem to be in conflict.  Why not just "update3"?

Comment (tannergooding) [inline netci.groovy]: Carry-over from when we had a static machine pool. A proper name would probably be 'auto-update3', but I opted for following the current naming convention.

**Comment by tannergooding** [comment]
Merging. All jobs are passing: http://dotnet-ci.cloudapp.net/job/dotnet_roslyn/job/master/job/GenPRTest/

**Pull request merged**